pkg/controller/plan/adapter/converter.go

```diff
@@ -186,11 +192,6 @@ func makeConversionContainer(pvc *v1.PersistentVolumeClaim, srcFormat, dstFormat
 		Image: base.Settings.VirtV2vImageCold,
 		SecurityContext: &v1.SecurityContext{
 			AllowPrivilegeEscalation: ptr.To(false),
-			RunAsNonRoot:             ptr.To(true),
-			RunAsUser:                ptr.To(int64(107)),
```

Comment:
did you check that it works on k8s?

Reply:
I've added Settings.OpenShift in #776  - that way we can differentiate between OpenShift and non-OpenShift clusters and specify RunAsNonRoot and RunAsUser on non-OpenShift clusters (I don't remember if I tried to set RunAsNonRoot=true without settings RunAsUser, I think I did and it failed)

Reply:
ok, this scenario wouldn't work on the default namespace because even though the pod starts, the conversion would fail to get write permissions on the PV so the settings of the VDDK validator is not the best reference for this